### ISSUE TITLE
Ease of training on Color images  ?

### ISSUE THREAD

**Issue by navneeth** [cwkx/GON#2]
Hello,
Thank you for your innovation & sharing it with others.  I see that all the training codes use single channel image datasets. (though a cifar10 result is shown in the paper).
Is it is as easy to train it on 3 channel (color images) ? Or are there any caveats ?
Please share your know how.

thanks !

**Comment by samb-t** [comment]
Hi Navneeth,
Thanks for your interest! From our experiments GONs work just as well with colour images: they are just as easy to train and we are not aware of any caveats. In fact, all graphs in our ICLR submission are on CIFAR-10 and we show that most of the time GONs achieve lower reconstruction loss in fewer epochs compared to standard autoencoders despite needing less than half the number of parameters. You just need to switch the dataset and change nc (number of channels) to 3 on line 13 in GON.py.
![image](https://user-images.githubusercontent.com/18750296/98232760-f8b19180-1f55-11eb-911f-5adc6e5991b4.png)

**Comment by navneeth** [comment]
Excellent ! I am eager to try it out
